cocos/2d/CCNode.h

```diff
@@ -1862,8 +1866,13 @@ class CC_DLL Node : public Ref
 #if CC_USE_PHYSICS
     PhysicsBody* _physicsBody;
 public:
-    void setPhysicsBody(Component* physicsBody) 
-    { 
+    void setPhysicsBody(PhysicsBody* physicsBody)
```

Comment:
@minggo because `setPhyiscsBody `should only be called with a physicsbody, and if i remember correctly, I can't call removeComponent with just `PhysicsBody `forward declared.